moveit_ros/planning/moveit_cpp/src/planning_component.cpp

```diff
@@ -398,25 +376,7 @@ bool PlanningComponent::setGoal(const std::string& goal_state_name)
 
 bool PlanningComponent::execute(bool blocking)
 {
-  if (!last_plan_solution_)
-  {
-    RCLCPP_ERROR(LOGGER, "There is no successful plan to execute");
-    return false;
-  }
-
-  // TODO(henningkayser): parameterize timestamps if required
-  // trajectory_processing::TimeOptimalTrajectoryGeneration totg;
-  // if (!totg.computeTimeStamps(*last_solution_trajectory_, max_velocity_scaling_factor_,
-  // max_acceleration_scaling_factor_))
-  //{
-  //  RCLCPP_ERROR("Failed to parameterize trajectory");
-  //  return false;
-  //}
-  return moveit_cpp_->execute(last_plan_solution_.trajectory_, blocking);
-}
-
-const planning_interface::MotionPlanResponse& PlanningComponent::getLastMotionPlanResponse()
-{
-  return last_plan_solution_;
+  RCLCPP_ERROR(LOGGER, "There is no successful plan to execute");
```

Comment:
I am wondering whether or not this means we can remove the whole function :thinking:

Reply:
I would say so

Reply:
Prob should add a deprecation warning